src/Enum/EconomicStrategyStatus.ts

```diff
@@ -3,5 +3,7 @@ export enum EconomicStrategyStatus {
   INSUFFICIENT_BALANCE = 'Not enough balance to claim.',
   CLAIM = 'Transaction can be claimed.',
   DEPOSIT_TOO_HIGH = 'The transaction deposit is too high.',
-  WINDOW_TOO_SHORT = 'Execution window is too short'
+  WINDOW_TOO_SHORT = 'Execution window is too short',
```

Comment:
I would change this to `TOO_SHORT_EXECUTION_WINDOW` to match the rest of the `TOO_SHORT_X_WINDOW` entries.

Reply:
Yeah but the only place it was used in has been removed in this PR. Unit tests should also reflect this change then.